src/components/common/ColonyHome/ColonyMembersWidget/ColonyMembersWidget.tsx

```diff
@@ -33,10 +36,20 @@ const ColonyMembersWidget = ({
   maxAvatars,
 }: Props) => {
   const { colony } = useColonyContext();
+  if (!colony) return null;
+  // console.log(`🚀 ~ colony`, colony);
 
-  if (!colony) {
-    return null;
-  }
+  // const { data, loading } = useGetMembersForColonyQuery({
+  //   skip: !colony?.colonyAddress,
+  //   variables: {
+  //     input: {
+  //       colonyAddress: colony?.colonyAddress ?? '',
+  //     },
+  //   },
+  //   fetchPolicy: 'cache-and-network',
+  // });
+
+  // console.log(`🚀 ~ data`, data);
```

Comment:
Are these debug comments still needed ?

Reply:
I'm working on ColonyMembers & Subsection files in a different PR, and cherry picked some commits from that PR. This brought a few other updates. I didnt waste time on these files as they will be changed very shortly.